core-bundle/contao/controllers/BackendMain.php

```diff
@@ -247,8 +247,9 @@ protected function compileTemplateData(array $data): array
 		$twig = $container->get('twig');
 
 		$data['menu'] = $twig->render('@Contao/backend/chrome/main_menu.html.twig');
-		$data['headerMenu'] = $twig->render('@Contao/backend/chrome/header_menu.html.twig');
-		$data['searchEnabled'] = System::getContainer()->getParameter('contao.backend_search.enabled');
+		$data['headerMenu'] = $twig->render('@Contao/backend/chrome/header_menu.html.twig', [
+			'searchEnabled' => System::getContainer()->getParameter('contao.backend_search.enabled')
```

Comment:
We should now remove the `ontao.backend_search.enabled` parameter and use 
`System::getContainer()->has('contao.search.backend') && System::getContainer()->get('contao.search.backend')->isAvailable()`.

The code that sets the parameter in the extension should also be removed :)

Reply:
You mean removing these lines, right?

https://github.com/contao/contao/blob/000a3dceb4beb16f8d01712545feb5d605516c70/core-bundle/src/DependencyInjection/ContaoCoreExtension.php#L341-L346